ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRao.java

```diff
@@ -52,27 +59,68 @@ public String getVersion() {
         return "1.0.0";
     }
 
+    private SystematicSensitivityAnalysisResult runSensiFallback(Network network,
```

Comment:
Rename runSensiWithFallback ? When I read I felt like this was only the fallback calculation but it's not